spec/oas3.patched/components/schemas/Plan.yaml

```diff
@@ -94,10 +94,6 @@ properties:
               type: integer
             type:
               type: string
-              enum:
```

Comment:
Should we offer these values as a description?

Reply:
I see no utility in adding them as a description in an SDK-local patch, especially since they aren't even the complete set of possible values for the property.  Certainly if these properties are converted to plain strings upstream (my personal preference, since the enums aren't used/needed to differentiate between types), I think it would be good to include a description of the likely values.